<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRegion</class>
 <widget class="QDialog" name="EditRegion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>241</width>
         <height>37</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;X: Mesafe , Y:Derinlik&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_X1">
              <property name="minimumSize">
               <size>
                <width>87</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>X1:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="buddy">
               <cstring>lineEdit_X1</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_X2">
              <property name="minimumSize">
               <size>
                <width>87</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>X2:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="buddy">
               <cstring>lineEdit_X2</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_Y">
              <property name="minimumSize">
               <size>
                <width>87</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>  Y:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="buddy">
               <cstring>lineEdit_Y</cstring>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLineEdit" name="lineEdit_X1">
              <property name="minimumSize">
               <size>
                <width>142</width>
                <height>25</height>
               </size>
              </property>
              <property name="maxLength">
               <number>15</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_X2">
              <property name="minimumSize">
               <size>
                <width>142</width>
                <height>25</height>
               </size>
              </property>
              <property name="maxLength">
               <number>15</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_Y">
              <property name="minimumSize">
               <size>
                <width>142</width>
                <height>25</height>
               </size>
              </property>
              <property name="maxLength">
               <number>15</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="edit">
       <property name="minimumSize">
        <size>
         <width>241</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>DÃ¼zenle</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
